app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -14,6 +14,7 @@ import DateInput from '../../util/DateInput';
 import confirmDialog from '../../util/ConfirmDialog';
 import { advancedFilterOptions } from '../../../data/advancedFilterOptions';
 import { getAllLanguageDisplayNames } from '../../../utils/Languages';
+import { getVaccineAdvancedFilters } from '../../../utils/VaccineAdvancedFilters';
```

Comment:
just a heads up, @sgober is refactoring how advanced filters are saved so depending on which PR gets merged first, the other will have merge conflicts